### PR TITLE
Statuses service for the Material Demand and Capacity Groups - 67

### PR DESCRIPTION
**Pull request by sajaadem**
## Description
Added the Statuses service for the Material Demand and Capacity Groups

## Pre-review checks

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by carslen** [comment]
Is this the same as "Feature/dcmfoss 67 copy"? Please use more unique PR titles ;-)

Please resolve merge conflict.

**Comment by sajaadem** [comment]
> Is this the same as "Feature/dcmfoss 67 copy"? Please use more unique PR titles ;-)
> 
> Please resolve merge conflict.


@carslen  I have closed the other similar pull requests and that due to a problem on the used username which caused issues in the checks to fail. 

So Could you review this pull requests? I have resolved the conflicts.

**Review by Ruskyy — APPROVED** [review]
Everything seems to be in line with the feature

File: demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/repositories/StatusesRepository.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package org.eclipse.tractusx.demandcapacitymgmt.demandcapacitymgmtbackend.repositories;
```
Comment (carslen) [inline demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/repositories/StatusesRepository.java:1]: Licence and copyright header missing. Please add.

**Review by carslen — CHANGES_REQUESTED** [review]

File: demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/repositories/StatusesRepository.java (hunk 1 above)
Comment (sajaadem) [reply demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/repositories/StatusesRepository.java:24]: @carslen Done

**Comment by carslen** [comment]
Please monitor open pull requests actively. I'm going to handle the PRs chronologically.

**Review by carslen — APPROVED** [review]

**Pull request merged**